src/components/allComments.js

```diff
@@ -80,22 +82,20 @@ class AllComments extends React.Component {
           ) : (
             <div width="36" height="36" />
           )
-        const appLabel =
-          apps.length === 1 ? <>{apps[0].node.name}</> : <>{c.object}</>
-        const appLink =
-          apps.length === 1 ? `/appco/${apps[0].node.appcoid}/#comments` : c.object
-        const rating = (
-          <>
+        const appLabel = apps.length === 1
+            ? `${apps[0].node.name}`
+            : `${c.object}`
+        
+        const rating = <>
             <br />
             <SmallRating component="span" readOnly value={c.rating} />
           </>
-        )
         const comment = c.comment.toString()
         comments.push(
           <ListItem
             button
             key={c._id}
-            onClick={() => this.handleClick(appLink)}
+            onClick={() => this.handleClick(c)}
```

Comment:
We want to use a different link instead of c.object.

Reply:
what different link?

Reply:
to the app deatils, see in the code above.